src/utils/get-command-spec.ts

```diff
@@ -238,6 +240,10 @@ export default async function getCommandSpec(directory: string): Promise<Generic
 			spec.data.accepts = arrayOrPromise instanceof Promise ? await arrayOrPromise : arrayOrPromise;
 		}
 
+		if ((spec.data?.acceptsMultiple || spec.data?.accepts) && !(spec.data?.accepts instanceof Array)) {
+			throw new Error('data.accepts must resolve to an Array');
```

Comment:
I don't think "array" is a proper noun here:

```suggestion
			throw new Error('data.accepts must resolve to an array');
```